examples/basicauth/main.go

```diff
@@ -34,14 +35,14 @@ func main() {
 	}
 
 	client := github.NewClient(tp.Client())
-	user, _, err := client.Users.Get("")
+	user, _, err := client.Users.Get(context.Background(), "")
 
 	// Is this a two-factor auth error? If so, prompt for OTP and try again.
 	if _, ok := err.(*github.TwoFactorAuthError); err != nil && ok {
 		fmt.Print("\nGitHub OTP: ")
 		otp, _ := r.ReadString('\n')
 		tp.OTP = strings.TrimSpace(otp)
-		user, _, err = client.Users.Get("")
+		user, _, err = client.Users.Get(context.Background(), "")
```

Comment:
I don't think we want to show calling `context.Background` twice in a row within a single function as an example to be followed. Instead, please assign `ctx := context.Background()` once before or after line 37 and then use `ctx` in both locations in place of `context.Background()`.